payment/src/test/java/com/yas/payment/service/PaymentProviderServiceTest.java

```diff
@@ -37,6 +74,63 @@ void setUp() {
         paymentProvider.setEnabled(true);
     }
 
+    @Test
+    @DisplayName("Create Payment Provider successfully")
+    void createPaymentProvider() {
+        // Given
+        var randomVal = UUID.randomUUID().toString();
```

Comment:
I noticed some variable declarations using "var" with names like cod and randomVal that could be a bit unclear. Since "var" already skips the type, would it make sense to use more descriptive names that align with the current context? I think this could improve readability. For example, in this case, we could change it to randomId or sampleId. What do you think?

Reply:
Regarding the name of `randomVal`, I tend create a random value to test the mapper, so just set that value for all `String.class` fields.
```java
private static @NotNull CreatePaymentVm getCreatePaymentVm(String randomVal) {
     CreatePaymentVm createPaymentVm = new CreatePaymentVm();
     createPaymentVm.setId(randomVal);
     createPaymentVm.setEnabled(true);
     createPaymentVm.setName(randomVal);
     createPaymentVm.setConfigureUrl(randomVal);
     createPaymentVm.setLandingViewComponentName(randomVal);
     return createPaymentVm;
}
```
Therefore, the name `randomVal` is make sense as that is what I meant to.

Reply:
About the `cod`, It is stand for `Cash On Delivery` and it's common term. So I wonder what make you concern on it?

Reply:
Thank you for the clarification! I understand your points regarding both `randomVal` and `cod`.

For `randomVal`, your explanation makes sense, especially in the context of testing the mapper. I don’t have any further concerns there.

For `cod`, I see your reasoning as well. Since it's a common term, it can be clear in many cases. My only thought is that it might not be immediately apparent whether it represents a provider or something else related to COD when scanning the code. So, when reading the code where `cod` is used, I find myself needing to recall that it refers to a provider.

That said, I respect your view, and this might just come down to personal preference.

Thanks again for taking the time to explain.

Reply:
Oh, I got your concern, it will make us confused if the term `cod` appears on `media` module.

However, this case we are in `com.yas.payment.service.MediaServiceTest`, so the term `cod` is ok here, we all know what is `cod` in payment context 😉.